README.md

```diff
@@ -1,4 +1,20 @@
 Put the time in weeks.  We'll change the format as need be.  I'm too busy
 putting my kid to sleep to do this nicely ;)
 
+** Current Format Requirements **
+Column and Type:
+
+* Company [string] - Company Name Or Governing Body
+* Paternity [int] - Number of Weeks Paid Paternity Leave
+* Maternity [int] - Number of Weeks Paid Maternity Leave
+* Adoption [int] - Number of Weeks Paid Leave Following an Adoption
+* Notes [string] - Any additional information of note
+* Source [string] - Document Source if Available 
+* Paid/Unpaid [string] - Either 'Paid' or 'Unpaid' as appropriate
+* Policy_Creation_Date [date] - Date of Creation (year, include month and day if possible)
+* Policy_Last_Updated [date] - Date of Last Policy Update (year, include month and day if possible)
+* Date_Added_to_Repo [date] - Date Stamp the record was added to this registry.
```

Comment:
If we move to the form instead of a directly modified CSV the date added to the form may not be the date added to the repo